commercetools-models/src/test/java/io/sphere/sdk/subscriptions/SnsUtils.java

```diff
@@ -0,0 +1,40 @@
+package io.sphere.sdk.subscriptions;
+
+import com.amazonaws.services.sns.AmazonSNS;
+import com.amazonaws.services.sns.model.CreateTopicResult;
+
+import java.util.UUID;
+
+/**
+ * Util methods for working with aws sns.
+ */
+public class SnsUtils {
+    private SnsUtils() {
+    }
+
+    /**
+     * Creates a new sns test topic.
+     *
+     * @param snsClient the sqs client
+     * @return the arn of the created sns topic
```

Comment:
It took me a while to realize that ARN was an acronym for some AWS stuff. Could we write it uppercase in the Javadoc to leave it clear it's not a word? Same with SQS and SNS.